test/test_transforms_v2_functional.py

```diff
@@ -417,22 +415,6 @@ def test_pil_output_type(self, info, args_kwargs):
 
         assert isinstance(output, PIL.Image.Image)
 
-    @make_info_args_kwargs_parametrization(
-        DISPATCHER_INFOS,
-        args_kwargs_fn=lambda info: info.sample_inputs(),
-    )
-    def test_dispatch_datapoint(self, info, args_kwargs, spy_on):
```

Comment:
Oops, I need to reinstate that. The refactored v2 tests have an updated version of this.